src/store/store.ts

```diff
@@ -126,24 +125,9 @@ export const useStore = defineStore('store', {
       }
     },
 
-    /** Whether the user has "staff" auth role. */
+    /** Whether the user has "staff" Keycloak role. */
     isRoleStaff (): boolean {
-      return this.getTombstone.authRoles.includes('staff')
```

Comment:
I got rid of `getTombstone` as it was only used in here and, in my opinion, complicated the understanding of where the data came from -- compare with new code which is now consistent across getters >>>